rfc/src/intro.md

```diff
@@ -36,6 +36,8 @@ This is the overall workflow for the RFC process:
    2. Start from a fork of the Kani repository.
    3. Copy the template file ([`rfc/src/template.md`](./template.md)) to `rfc/src/rfcs/<ID_NUMBER><my-feature>.md`.
    4. Fill in the details according to the template instructions.
+     - For the first RFC version, we recommend that you leave the "Software Design" section empty.
+       Focus on the user impact and user experience.
```

Comment:
I would add "Include a few examples if possible".

Reply:
> I would add "Include a few examples if possible".

Where would you add that? We could add that sentence to the template itself. I'm assuming you are talking about the UX part, right?

Reply:
> Where would you add that? 

To this sentence, i.e. `Focus on the user impact and user experience. Include a few usage examples if possible.`.